samcli/commands/check/resources/LambdaFunctionPricing.py

```diff
@@ -0,0 +1,17 @@
+"""
+Object to contain Lambda function pricing info.
+This data gets stored in the graph, not a lambda function object
```

Comment:
the comment added here should be related to the module itself, not the class or objects